src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int colorNumber = new Random().nextInt(Color.values().length);
+        return Color.valueOf(Color.values()[colorNumber].name());
```

Comment:
```suggestion
        return Color.values()[colorNumber]
```
You can make it easier